keps/prod-readiness/sig-network/1880.yaml

```diff
@@ -0,0 +1,5 @@
+kep-number: 1880
+alpha:
+  approver: "@wojtek-t"
+beta:
```

Comment:
Please remove beta for now :)

Also ping me once you will have overall agreement with SIG leads.